gitbook_docs/summary.md

```diff
@@ -77,13 +77,13 @@
 * User Management
   * Resources
     * [Group](../docs/resources/group.md)
-    * [S3 Key](../docs/resources/s3_key.md)
+    * [Key](../docs/resources/s3_key.md)
```

Comment:
Since this is under User Management, Object Storage Key would be less ambiguous as to what the key relates to.